### PR TITLE
Ignition 7.9 dev2

### PR DESCRIPTION
**Pull request by TomKimsey**
## Major Changes
* Added write functionality to Ignition 7.9

## Minor Changes
* correct url encoding
* whitespace fixes
* added changelog

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,3 +5,61 @@ Synchronize DataMailbox data to Ignition's Tag Historian
...
+
+Example
+```
+# Java Additional Parameters
+wrapper.java.additional.1=-XX:+UseConcMarkSweepGC
+wrapper.java.additional.2=-Ddata.dir=data
+wrapper.java.additional.3=-Dignition.allowunsignedmodules=true
+```
+
+Once the ignition.conf file has been edited save and close the file, then restart igntion.
```
Comment (TomKimsey) [inline readme.md]: Ignition is spelled wrong, I will fix that in the next push.

File: eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java [hunk 2]
```diff
@@ -116,6 +116,23 @@ public EwonsData syncData(Long transactionId) throws Exception {
...
+      params.add(EwonConsts.T2M_PARAM_TAGNAME1);
+      params.add(tagName);
+
+      params.add(EwonConsts.T2M_PARAM_TAGVALUE1);
+      params.add(tagValue);
+
+      params.add(EwonConsts.T2M_M2W_DEVKEY);
+      params.add(authInfo.getDevKey());
+
+      EwonUtil.httpGet(buildT2MCall(directory, EwonConsts.T2M_CALL_UPDATETAGFORM, params.toArray(new String[params.size()])));
```
Comment (alexjhawk) [inline eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java]: Probably just my personal preference, but for lines that have a lot of nested parentheses, it's easier to read and understand with some spaces. 

EwonUtil.httpGet( buildT2MCall( directory, EwonConsts.T2M_CALL_UPDATETAGFORM, params.toArray( new String[ params.size() ] ) ) );

Comment (TomKimsey) [reply eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java]: thats a tough one, i dont think google style has strict rules on that.  I use a bracket colorizer in my editor which color codes the brackets for each level.  I can change it if it helps people with other editors.

Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java]: I don't see strict rules either. I'm not sure other opinions, but that'ss just my preference. I did just get a bracket colorizer though, so its not a big deal.

File: readme.md (hunk 1 above)
Comment (TomKimsey) [reply readme.md]: Fixed

**Pull request merged**